models.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+
+const { Schema, model, Types } = mongoose;
+const { ObjectId } = Types;
+
+// schemas definition
+const categoryScheme = new Schema({
+  _id: ObjectId,
```

Comment:
@IgorChaikin I thinks you can skip `_id` field description, it will be added automatically by `mongoose`, also could you add `required: true`, to some required fields, like `title`, or `product.price`

```js
const productSchema = new Schema({
  name: { type: String, required: true },
  image: { type: String, required: true },
  price: { type: String, required: true },
  description: String,
})
```